### DISCUSSION TITLE
Network sensor nodes and broadcast communication.

### DISCUSSION THREAD

**Discussion by husnain-ce** [omnetpp/omnetpp#1035] [Q&A]
Basically the scenario is there a network of sensor nodes and satellites that is established. When the satellite reaches the sensor nodes, it sends a beacon to notify them of its availability. Then sensors should have a predefined algorithm for time and frequency allocation to start sending their data to the satellite. I have matrix containing the sensor indexes, priorities (some with higher priority for allocation), and the amount of data to be sent. Using this matrix, the sensors will efficiently send their data to the satellite.

**Comment by tabgab** [comment]
And your question is?

**Reply by husnain-ce** [reply]
I have code in C++ want to integrate this into Omentt++, would like to know the learning roadmap or guide so I can proceed.
